msal/src/main/java/com/microsoft/identity/client/Logger.java

```diff
@@ -138,10 +138,6 @@ public synchronized void setExternalLogger(final ILoggerCallback externalLogger)
             return;
         }
 
-        if (null != mExternalLogger) {
-            throw new IllegalStateException("External logger is already set, cannot be set again.");
-        }
-
```

Comment:
Why do we have a reference to this mExternalLogger if it isn't used anywhere but inside this function?

Reply:
yeah no idea....I can remove it

Reply:
Addressed